apps/user-profile/src/components/modules/UserOnboardingModal/PlatformOnboarding/PageOne/PageOne.js

```diff
@@ -1,7 +1,7 @@
 import { Typography } from "@devlaunchers/components/components/atoms";
 import "/Users/rinabo/developments/dev-launchers-platform/apps/user-profile/src/images/logo-monogram.png" 
```

Comment:
I think this import causes the app to break, because you're importing things from your device rather than relative paths. 
always importing with relative paths like in line 3